src/TodoList.js

```diff
@@ -6,12 +6,14 @@ import TodoListItem from './TodoListItem';
 
 function TodoList ({todoList}) {
  
-  const {id, title} = todoList;
+  
 
   return (
     <ul>   
       {todoList.map(function(item){
-        return (<TodoListItem key={id} item={item} />);
+        const {id, title} = item;
```

Comment:
nice enhancement destructuring the item properties. I would suggest to dele the console log. :)